tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph_executions/graph_executions_component.css

```diff
@@ -67,24 +67,34 @@ limitations under the License.
   border-bottom: 1px solid rgba(0, 0, 0, 0.12);
   display: flex;
   flex-wrap: nowrap;
-  height: 36px;
-  line-height: 36px;
+  height: 38px;
+  line-height: 38px;
   text-align: left;
   vertical-align: middle;
   white-space: nowrap;
   width: 100%;
 }
 
 .tensor-name {
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
   direction: rtl;
   display: block;
   height: 16px;
   line-height: 16px;
+  margin: 2px 0 1px;
+  max-width: calc(100% - 3px);
```

Comment:
`box-sizing: border-box;` tho :(

Reply:
Good point. Using `box-sizing: border-box;` is more appropriate here. Adde it.